lib/KFocusTrap.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+
+  <div>
+    <div
+      v-if="!disabled"
+      class="focus-trap-first"
+      tabindex="0"
+      @focus="handleFirstTrapFocus"
+    ></div>
+
+    <slot></slot>
+
+    <div
+      v-if="!disabled"
+      class="focus-trap-last"
+      tabindex="0"
+      @focus="handleLastTrapFocus"
+    ></div>
+  </div>
+
```

Comment:
@lokesh-sagi125 Try to remove this line above(position cursor there and press Backspace) and add it back(press Enter). It seems to do the trick for me. Looks like the linter for some reason doesn't agree with some empty space there.

Reply:
That should fix fix the linting issue!

Reply:
tried it , same issue. 
Could you review the rest of the code and let me know if there are any changes that should be made? Thank you!

Reply:
Is it okay iff I fix linting issue so you are unblocked?

Reply:
yes @akolson  id be glad.

Reply:
but, is it just the line spacing thats causing the linting issue or is there a hidden cause that i'm missing?

Reply:
Fixed.

For reference, this is all I did,
> @lokesh-sagi125 Try to remove this line above(position cursor there and press Backspace) and add it back(press Enter). It seems to do the trick for me. Looks like the linter for some reason doesn't agree with some empty space there.

Reply:
thank you @akolson  ill keep this mind for future contributions.